ntbs-service/Pages/PreviousHistoryDetails/Edit.cshtml

```diff
@@ -0,0 +1,64 @@
+@page "{handler?}"
+@model ntbs_service.Pages_PatientTBHistory.EditModel
+@using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
+
+@{
+    ViewData["Title"] = "Notification - Patient TB History";
+}
+@{string errorClassName = "nhsuk-form-group--error";}
+<div><a asp-page="/Patients/Index">Back to List</a></div>
+
+<h2>Previous History Details for New Patient</h2>
+
+<div class="row">
+    <date-comparison inline-template>
+        <div class="col-md-5">

```

Comment:
As per my bootstrap-removing changes on your other branch, I'd just kick this and the `row` divs out for now